test/lib/jdk/test/lib/util/FileUtils.java

```diff
@@ -377,6 +377,35 @@ public static long getProcessHandleCount() {
         }
     }
 
+    /**
+     * Patches a part of a file.
+     * @param path of file
+     * @param fromLine the first line to patch. This is the number you see in an editor, 1-based.
```

Comment:
Perhaps this should mention whether the `fromLine` is inclusive, like it's noted for the `toLine`?